web_services/foxx/brightid/db.js

```diff
@@ -922,13 +923,18 @@ function isEthereumAddress(address) {
   return re.test(address);
 }
 
-function getAppUserIds(appKey, activeOnly, countOnly) {
+function getAppUserIds(appKey, period, countOnly) {
   const app = getApp(appKey);
   const res = [];
   let query = { app: app._key };
-  if (activeOnly) {
+  if (period == "current") {
     const vel = app.verificationExpirationLength;
     query["roundedTimestamp"] = vel ? parseInt(Date.now() / vel) * vel : 0;
+  } else if (period == "previous") {
+    const vel = app.verificationExpirationLength;
+    query["roundedTimestamp"] = vel
+      ? (parseInt(Date.now() / vel) - 1) * vel
+      : 0;
   }
   for (const verification of app.verifications) {
     query["verification"] = verification;
```

Comment:
I could not place comment in this bellow section:
```
const appUserIds = appIdsColl
      .byExample(query)
      .toArray()
      .map((d) => d.appId);```
I think we should create a `Set` here.

Reply:
https://github.com/BrightID/BrightID-Node/commit/4b2d386eea24a0cbbdfaa7054d5b90e94a5cdf08